localnet/ethermint/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+DATA_DIR=/ethermint
+CONFIG=$DATA_DIR/config/config.toml
+
+sed -i 's/prometheus = false/prometheus = true/g' $CONFIG
```

Comment:
We don't have much of the same GH actions on here as on Evmos/Ethermint repos, but this would be flagged there in `shellcheck`
```suggestion
sed -i 's/prometheus = false/prometheus = true/g' "$CONFIG"
```